app/services/migration/user_migrator.rb

```diff
@@ -0,0 +1,117 @@
+module Migration
+  class UserMigrator
+    attr_reader :ecf_user
+
+    def initialize(ecf_user)
+      @ecf_user = ecf_user
+    end
+
+    def find_initialize_or_merge_user
```

Comment:
I think this could just be `find_or_initialize`; the merging that goes on is an implementation detail and having it in the method name makes it unclear if it will return anything if merging

Reply:
renamed👍